### PR TITLE
[com_content]php and xml code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for com_content
### Testing Instructions

code review

cc @andrepereiradasilva @JoomliC

### CODE REVIEW

File: administrator/components/com_content/config.xml [hunk 1]
```diff
@@ -62,54 +68,58 @@
...
 
 		<field
 			name="link_parent_category"
 			type="radio"
-			class="btn-group btn-group-yesno"
 			label="JGLOBAL_LINK_PARENT_CATEGORY_LABEL"
 			description="JGLOBAL_LINK_PARENT_CATEGORY_DESC"
+			class="btn-group btn-group-yesno"
 			default="1"
 			showon="show_parent_category:1">
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: new line for <code>></code>

File: administrator/components/com_content/config.xml [hunk 2]
```diff
@@ -422,10 +461,10 @@
...
-				<option value="1">JBROWSERTARGET_NEW</option>
-				<option value="2">JBROWSERTARGET_POPUP</option>
-				<option value="3">JBROWSERTARGET_MODAL</option>
+			<option value="0">JBROWSERTARGET_PARENT</option>
+			<option value="1">JBROWSERTARGET_NEW</option>
+			<option value="2">JBROWSERTARGET_POPUP</option>
+			<option value="3">JBROWSERTARGET_MODAL</option>
 		</field>
 		<field
 			name="spacer4"
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: indent for closing tag <code>/></code>

File: administrator/components/com_content/config.xml [hunk 3]
```diff
@@ -440,83 +479,94 @@
...
 		/>
+
 		<field
 			name="show_category_heading_title_text"
 			type="radio"
-			class="btn-group btn-group-yesno"
  			label="JGLOBAL_SHOW_CATEGORY_HEADING_TITLE_TEXT_LABEL"
 			description="JGLOBAL_SHOW_CATEGORY_HEADING_TITLE_TEXT_DESC"
+			class="btn-group btn-group-yesno"
 			default="1">
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: new line for <code>></code>

File: administrator/components/com_content/config.xml [hunk 4]
```diff
@@ -440,83 +479,94 @@
...
-				<option value="0">JHIDE</option>
+			<option value="1">JSHOW</option>
+			<option value="0">JHIDE</option>
 		</field>
-		<field name="show_category_title"
+
+		<field
+			name="show_category_title"
 			type="radio"
 			class="btn-group btn-group-yesno"
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: Class below...

File: administrator/components/com_content/config.xml [hunk 5]
```diff
@@ -611,99 +670,111 @@
...
 			type="text"
-			default="1"
 			label="JGLOBAL_NUM_LEADING_ARTICLES_LABEL"
-			description="JGLOBAL_NUM_LEADING_ARTICLES_DESC">
-		</field>
+			description="JGLOBAL_NUM_LEADING_ARTICLES_DESC"
+			default="1"
+		/>
 
 		<field name="num_intro_articles"
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: new line for name!

File: administrator/components/com_content/config.xml [hunk 6]
```diff
@@ -611,99 +670,111 @@
...
-		</field>
+			description="JGLOBAL_NUM_COLUMNS_DESC"
+			default="2"
+			/>
 
-		<field name="num_links"
+		<field
+			name="num_links"
 			type="text"
 			default="4"
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: default after description please

File: administrator/components/com_content/config.xml [hunk 7]
```diff
@@ -931,14 +1012,14 @@
 		label="JCONFIG_PERMISSIONS_LABEL"
 		description="JCONFIG_PERMISSIONS_DESC"
 		>
-
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: isn't it better readibility if always an empty line before each opening tag ? (just a question about your opinion on this ;-) )

**Comment by JoomliC** [comment]
@wojsmol I only checked config.xml, and already 6 errors found...
Could you please do a new checking of all yours changes in this PR ? (i know it's time, but it could save time too on review ;-) )

One question: you started with plugins, then modules, and now components, but not all modules/plugins are reviewed (just did mod_multilangstatus yesterday in the same time of another purpose), so, is there a reason for that ? (code style review is a long and not funny job, but good to be done, but maybe i could help later when i will have finished some other work on modals) ;-) ).
Note: could you not work on com_templates for now, as i have started many changes in many files, and so, i can do the code style review for this component in the same time)

**Comment by wojsmol** [comment]
@JoomliC I reviewed my changes again. At the moment they are reviewed all the frontend modules and plugins. PR for com_content I have prepared a little earlier than planned in connection with my discussion with JM in PR https://github.com/joomla/joomla-cms/pull/10621.

File: administrator/components/com_content/config.xml [hunk 8]
```diff
@@ -611,99 +673,112 @@
...
-		</field>
+			description="JGLOBAL_NUM_LEADING_ARTICLES_DESC"
+			default="1"
+		/>
 
-		<field name="num_intro_articles"
+		<field
+			name="num_intro_articles"
 			type="text"
 			default="4"
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/config.xml]: default should go down.

File: administrator/components/com_content/config.xml [hunk 9]
```diff
@@ -715,186 +790,198 @@
...
+			<option value="ralpha">JGLOBAL_TITLE_REVERSE_ALPHABETICAL</option>
+			<option value="author">JGLOBAL_AUTHOR_ALPHABETICAL</option>
+			<option value="rauthor">JGLOBAL_AUTHOR_REVERSE_ALPHABETICAL</option>
+			<option value="hits">JGLOBAL_MOST_HITS</option>
+			<option value="rhits">JGLOBAL_LEAST_HITS</option>
+			<option value="order">JGLOBAL_ARTICLE_MANAGER_ORDER</option>
+		</field>
+
+		<field
+			name="order_date" type="list"
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/config.xml]: type down

File: administrator/components/com_content/models/forms/article.xml [hunk 10]
```diff
@@ -1,161 +1,277 @@
...
-		<field name="created" type="calendar" label="COM_CONTENT_FIELD_CREATED_LABEL"
-			description="COM_CONTENT_FIELD_CREATED_DESC" size="22"
-			format="%Y-%m-%d %H:%M:%S" filter="user_utc" />
+		<field
+			name="created"
+			type="calendar"
+			label="COM_CONTENT_FIELD_CREATED_LABEL"
+			description="COM_CONTENT_FIELD_CREATED_DESC"
+			size="22"
+			format="%Y-%m-%d %H:%M:%S" filter="user_utc"
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/models/forms/article.xml]: filter down

File: administrator/components/com_content/models/forms/article.xml [hunk 11]
```diff
@@ -1,161 +1,277 @@
...
+		/>
 
-		<field name="publish_up" type="calendar"
-			label="COM_CONTENT_FIELD_PUBLISH_UP_LABEL" description="COM_CONTENT_FIELD_PUBLISH_UP_DESC"
+		<field
+			name="publish_up"
+			type="calendar"
+			label="COM_CONTENT_FIELD_PUBLISH_UP_LABEL"
+			description="COM_CONTENT_FIELD_PUBLISH_UP_DESC"
 			format="%Y-%m-%d %H:%M:%S" size="22"
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/models/forms/article.xml]: size down

File: administrator/components/com_content/models/forms/article.xml [hunk 12]
```diff
@@ -483,224 +627,260 @@
...
+			/>
 
 			<field
 				name="link_category"
 				type="hidden"
 				label="JGLOBAL_LINK_CATEGORY_LABEL"
-				description="JGLOBAL_LINK_CATEGORY_DESC">
-			</field>
+				description="JGLOBAL_LINK_CATEGORY_DESC"
+			>
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/models/forms/article.xml]: />

File: administrator/components/com_content/models/forms/article.xml [hunk 13]
```diff
@@ -483,224 +629,260 @@
...
+				description="JGLOBAL_SHOW_PUBLISH_DATE_DESC"
+			/>
 
 			<field
 				name="show_item_navigation"
 				type="hidden"
 				label="JGLOBAL_SHOW_NAVIGATION_LABEL"
-				description="JGLOBAL_SHOW_NAVIGATION_DESC">
-			</field>
+				description="JGLOBAL_SHOW_NAVIGATION_DESC"\
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/models/forms/article.xml]: remove the extra \

File: administrator/components/com_content/models/forms/filter_articles.xml [hunk 14]
```diff
@@ -56,29 +61,31 @@
 			<option value="">JOPTION_SELECT_LANGUAGE</option>
 			<option value="*">JALL</option>
 		</field>
+
 		<field
 			name="tag"
 			type="tag"
 			mode="nested"
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/models/forms/filter_articles.xml]: mode down

File: components/com_content/models/forms/article.xml [hunk 15]
```diff
@@ -131,242 +131,286 @@
...
-				description="COM_CONTENT_FIELD_URL_DESC" />
-			<field name="urlatext"
-				type="text"
-				label="COM_CONTENT_FIELD_URLA_LINK_TEXT_LABEL"
-				description="COM_CONTENT_FIELD_URL_LINK_TEXT_DESC"
-				class="inputbox"
-				size="20" />
+		</fieldset>
+	</fields>
+		<field
```
Comment (andrepereiradasilva) [inline components/com_content/models/forms/article.xml]: extra tab in this part?

Comment (andrepereiradasilva) [inline components/com_content/models/forms/article.xml]: no, it's missing the `<fields name="urls">`

File: components/com_content/models/forms/article.xml [hunk 16]
```diff
@@ -131,242 +131,286 @@
...
+			size="20"
+		/>
+
+		<field
+			name="targetc"
+			type="hidden"
+		/>
+	</fields>
+
+	<fields name="urls">
```
Comment (andrepereiradasilva) [inline components/com_content/models/forms/article.xml]: this should not be here. see previous comment

File: components/com_content/models/forms/article.xml [hunk 17]
```diff
@@ -131,242 +131,286 @@
...
+		<fields name="metadata">
+			<fieldset name="jmetadata" label="JGLOBAL_FIELDSET_METADATA_OPTIONS">
```
Comment (andrepereiradasilva) [inline components/com_content/models/forms/article.xml]: reduce tab in thsi two lines

File: components/com_content/views/archive/tmpl/default.xml [hunk 18]
```diff
@@ -61,30 +65,32 @@
...
 			</field>
 
-			<field name="introtext_limit" type="text" default="100"
-					label="JGLOBAL_ARCHIVE_ARTICLES_FIELD_INTROTEXTLIMIT_LABEL"
-					description="JGLOBAL_ARCHIVE_ARTICLES_FIELD_INTROTEXTLIMIT_DESC" />
-
+			<field
+				name="introtext_limit"
+				type="text"
+				default="100"
```
Comment (andrepereiradasilva) [inline components/com_content/views/archive/tmpl/default.xml]: default down

File: components/com_content/views/article/tmpl/default.xml [hunk 19]
```diff
@@ -31,258 +35,282 @@
...
-			label="JGLOBAL_SHOW_ICONS_LABEL"
-			description="JGLOBAL_SHOW_ICONS_DESC">
-			<option value="">JGLOBAL_USE_GLOBAL</option>
-			<option value="1">JSHOW</option>
-			<option value="0">JHIDE</option>
-		</field>
+			<field
+				name="show_print_icon"
+				type="radio"
+				class="btn-group"
```
Comment (andrepereiradasilva) [inline components/com_content/views/article/tmpl/default.xml]: class down

File: components/com_content/views/categories/tmpl/default.xml [hunk 20]
```diff
@@ -55,56 +62,65 @@
...
-</fieldset>
-
-<fieldset name="category" label="JGLOBAL_CATEGORY_OPTIONS">
-			<field name="spacer3" type="spacer" class="text"
-			label="JGLOBAL_SUBSLIDER_DRILL_CATEGORIES_LABEL"
+		<fieldset name="category" label="JGLOBAL_CATEGORY_OPTIONS">
+			<field
+				name="spacer3"
+				type="spacer"
+				class="text"
```
Comment (andrepereiradasilva) [inline components/com_content/views/categories/tmpl/default.xml]: class down

File: components/com_content/views/categories/tmpl/default.xml [hunk 21]
```diff
@@ -151,28 +173,34 @@
...
+		</fieldset>
 
-</fieldset>
-<fieldset name="blog" label="JGLOBAL_BLOG_LAYOUT_OPTIONS">
-				<field name="spacer4" type="spacer" class="text"
+		<fieldset name="blog" label="JGLOBAL_BLOG_LAYOUT_OPTIONS">
+			<field
+				name="spacer4"
+				type="spacer"
+				class="text"
```
Comment (andrepereiradasilva) [inline components/com_content/views/categories/tmpl/default.xml]: class down

File: components/com_content/views/categories/tmpl/default.xml [hunk 22]
```diff
@@ -357,167 +421,200 @@
...
 				<option value="0">JNo</option>
 				<option value="1">JYes</option>
 			</field>
 
-			<field name="show_author" type="list"
-				description="JGLOBAL_Show_Author_Desc"
+			<field
+				name="show_author"
+				type="list"
 				label="JGLOBAL_Show_Author_Label"
```
Comment (andrepereiradasilva) [inline components/com_content/views/categories/tmpl/default.xml]: All the language vars are uppercase.
See this whole block.

File: components/com_content/views/category/tmpl/default.xml [hunk 23]
```diff
@@ -256,11 +301,15 @@
...
 				<option value="0">JHIDE</option>
 				<option value="1">JSHOW</option>
-		</field>
-		<field name="display_num" type="list"
+			</field>
+
+			<field
+				name="display_num"
+				type="list"
 				default="10"
```
Comment (andrepereiradasilva) [inline components/com_content/views/category/tmpl/default.xml]: default down

**Comment by andrepereiradasilva** [comment]
ok please make the changes i identified and for me is ok.

File: components/com_content/models/forms/article.xml [hunk 24]
```diff
@@ -289,83 +290,83 @@
...
+	<fields name="urls">
+			<field
+				name="urla"
+				type="url"
+				label="COM_CONTENT_FIELD_URLA_LABEL"
+				description="COM_CONTENT_FIELD_URL_DESC"
+				validate="url"
+				filter="url"
+				relative="true"
+			/>
```
Comment (andrepereiradasilva) [inline components/com_content/models/forms/article.xml]: this whole block should be one tab less

Comment (wojsmol) [inline components/com_content/models/forms/article.xml]: fixed

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on c663e0608574dcf99373d266b2daa1da260bc653<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

File: administrator/components/com_content/config.xml [hunk 25]
```diff
@@ -3,10 +3,12 @@
 	<fieldset
 		name="articles"
 		label="JGLOBAL_ARTICLES"
-		description="COM_CONTENT_CONFIG_ARTICLE_SETTINGS_DESC">
+		description="COM_CONTENT_CONFIG_ARTICLE_SETTINGS_DESC"
+	>
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: I wonder if we didn't do an error (me too!) about indent here, as it's not the closing tag...
Maybe <code>></code> should be align with description, as it is for field when options ? (i will see soon to do some updates to the xml coding standards with already discussed rules ;-) )

Comment (wojsmol) [inline administrator/components/com_content/config.xml]: @JoomliC Thos is empty fild without options so see `Element is empty` example https://joomla.github.io/coding-standards/?coding-standards/chapters/xml.md

Comment (andrepereiradasilva) [inline administrator/components/com_content/config.xml]: i agree

File: administrator/components/com_content/config.xml [hunk 26]
```diff
@@ -4,7 +4,7 @@
 		name="articles"
 		label="JGLOBAL_ARTICLES"
 		description="COM_CONTENT_CONFIG_ARTICLE_SETTINGS_DESC"
-	>
+		>
```
Comment (andrepereiradasilva) [inline administrator/components/com_content/config.xml]: tab before, not after :D

Comment (andrepereiradasilva) [inline administrator/components/com_content/config.xml]: forget it, github fould me

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on ae4391038e8272f9a81431d00aa140bf1df0d613<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

**Comment by wojsmol** [comment]
@JoomliC Please test

File: administrator/components/com_content/views/article/tmpl/edit.php [hunk 27]
```diff
@@ -80,11 +80,11 @@
...
 // In case of modal
 $isModal = $input->get('layout') == 'modal' ? true : false;
-$layout = $isModal ? 'modal' : 'edit';
-$tmpl = $isModal ? '&tmpl=component' : '';
+$layout  = $isModal ? 'modal' : 'edit';
+$tmpl    = $isModal ? '&tmpl=component' : '';
 ?>
 
-<form action="<?php echo JRoute::_('index.php?option=com_content&layout=' . $layout . $tmpl . '&id=' . (int) $this->item->id); ?>" method="post" name="adminForm" id="item-form" class="form-validate">
+<form action="<?php echo JRoute::_('index.php?option=com_content&layout=' . $layout . $tmpl . '&id=' . (int) $this->item->id); ?>" method="post" id="item-form" class="form-validate" name="adminForm">
```
Comment (JoomliC) [inline administrator/components/com_content/views/article/tmpl/edit.php]: Could you explain this change ?!?

File: components/com_content/models/articles.php [hunk 28]
```diff
@@ -439,8 +439,8 @@ protected function getListQuery()
 				if ($authorAlias)
 				{
 					$type = $this->getState('filter.author_alias.include', true) ? 'IN' : 'NOT IN';
-					$authorAliasWhere = 'a.created_by_alias ' . $type . ' (' . $authorAlias .
-						')';
+					$authorAliasWhere = 'a.created_by_alias ' . $type . ' (' . $authorAlias
+						. ')';
```
Comment (JoomliC) [inline components/com_content/models/articles.php]: Why a return line ? Maybe could be only on one line...

File: components/com_content/models/articles.php [hunk 29]
```diff
@@ -472,24 +472,24 @@ protected function getListQuery()
...
 		{
 			case 'range':
 				$startDateRange = $db->quote($this->getState('filter.start_date_range', $nullDate));
-				$endDateRange = $db->quote($this->getState('filter.end_date_range', $nullDate));
+				$endDateRange   = $db->quote($this->getState('filter.end_date_range', $nullDate));
 				$query->where(
-					'(' . $dateField . ' >= ' . $startDateRange . ' AND ' . $dateField .
-						' <= ' . $endDateRange . ')'
+					'(' . $dateField . ' >= ' . $startDateRange . ' AND ' . $dateField
+						. ' <= ' . $endDateRange . ')'
```
Comment (JoomliC) [inline components/com_content/models/articles.php]: IMO, better the AND as start of the new line if 2 lines ;-)

File: components/com_content/views/category/tmpl/blog.xml [hunk 30]
```diff
@@ -24,304 +27,305 @@
...
+				label="JGLOBAL_NUM_COLUMNS_LABEL"
+				description="JGLOBAL_NUM_COLUMNS_DESC"
+				size="3"
+			/>
+
+			<field
+				name="num_links"
+				type="text"
+				description="JGLOBAL_NUM_LINKS_DESC"
+				label="JGLOBAL_NUM_LINKS_LABEL"
```
Comment (JoomliC) [inline components/com_content/views/category/tmpl/blog.xml]: Label before description

File: components/com_content/views/category/tmpl/default.xml [hunk 31]
```diff
@@ -256,11 +301,15 @@
 				<option value="">JGLOBAL_USE_GLOBAL</option>
```
Comment (JoomliC) [inline components/com_content/views/category/tmpl/default.xml]: show_pagination_results not reviewed...

File: components/com_content/views/category/tmpl/default.xml [hunk 32]
```diff
@@ -256,11 +301,15 @@
...
-		<field name="display_num" type="list"
-				default="10"
+			</field>
+
+			<field
+				name="display_num"
+				type="list"
 				description="JGLOBAL_NUMBER_ITEMS_LIST_DESC"
-				label="JGLOBAL_NUMBER_ITEMS_LIST_LABEL">
+				label="JGLOBAL_NUMBER_ITEMS_LIST_LABEL"
```
Comment (JoomliC) [inline components/com_content/views/category/tmpl/default.xml]: label before description...

**Comment by JoomliC** [comment]
@wojsmol Please, review all your files changed before ;-
Still some issues (and not have yet check all...)

Another question: is this PR supposed to be a full com_content code style review ?

File: administrator/components/com_content/views/article/tmpl/edit.php (hunk 27 above)
Comment (wojsmol) [inline administrator/components/com_content/views/article/tmpl/edit.php]: It was to be HTML CS, but now I see that done incorrectly, so restore the previous form.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on f8c31541fb3c27f2b5dbd8dc89d598e854941b6f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

File: administrator/components/com_content/views/article/tmpl/edit.php (hunk 27 above)
Comment (JoomliC) [inline administrator/components/com_content/views/article/tmpl/edit.php]: In fact, according to code standard (" _HTML attributes should be listed in an order that reflects the fact that class names are the primary interface through which CSS and JavaScript select elements_. ") : https://joomla.github.io/coding-standards/?coding-standards/chapters/html.md
It should be this : 
`<form class="form-validate" id="item-form" name="adminForm" action="<?php echo JRoute::_('index.php?option=com_content&layout=' . $layout . $tmpl . '&id=' . (int) $this->item->id); ?>" method="post"> `
More info here (Bootstrap standards) : http://codeguide.co/#html-attribute-order (seems logic)

In fact, order is better if in the logic of the page rendering : CSS, JS, ACTION, others (additional info, A11Y, ...)

Comment (wojsmol) [inline administrator/components/com_content/views/article/tmpl/edit.php]: This chenge was reverted in https://github.com/joomla/joomla-cms/pull/10659/commits/b1f0dea3b3e5ae74710ff85eb1b06f9f63c93c13. PHP CSsnoffer go crazy in view files and HTML CS was not the main purpose of this PR.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on d1796dcec0b78e0aba0c5899ad03bbe3ec6f8a02<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

**Comment by wojsmol** [comment]
@JoomliC I reviewd my chenges one more time.

**Comment by brianteeman** [comment]
@zero-24 @JoomliC Can you test this please and then we can get it merged<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

File: components/com_content/views/archive/tmpl/default.xml [hunk 33]
```diff
@@ -146,87 +158,101 @@
...
 				<option	value="0">JNO</option>
 				<option	value="1">JYES</option>
 			</field>
 
-			<field name="link_titles" type="list"
+			<field
+				name="link_titles"
+				type="list"
 				description="JGLOBAL_LINKED_TITLES_DESC"
 				label="JGLOBAL_LINKED_TITLES_LABEL"
```
Comment (JoomliC) [inline components/com_content/views/archive/tmpl/default.xml]: Label before description

File: components/com_content/views/form/tmpl/edit.xml [hunk 34]
```diff
@@ -1,29 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metadata>
-	<layout title="COM_CONTENT_FORM_VIEW_DEFAULT_TITLE" option="COM_CONTENT_FORM_VIEW_DEFAULT_OPTION">
+	<layout
+		title="COM_CONTENT_FORM_VIEW_DEFAULT_TITLE"
+		option="COM_CONTENT_FORM_VIEW_DEFAULT_OPTION"
+		>
 		<help
 			key = "JHELP_MENUS_MENU_ITEM_ARTICLE_CREATE"
```
Comment (JoomliC) [inline components/com_content/views/form/tmpl/edit.xml]: A reason sometimes on one line, and sometimes not ?

**Comment by JoomliC** [comment]
@brianteeman For changes done, it's almost good for me. (added 2 notes)

Additionnal comments : 
- PR has conflicts
- I didn't have checked not displayed lines if missing code review
- views files not reviewed (so not a complete component CS review) but this would need before to operate update of Travis, so just a mention, not a PR blocker.

I would say: fix branch conflicts and at least 1 note for label/desc order in xml and it's ok for me! ;-)

**Comment by brianteeman** [comment]
That rebase isnt correct unless you really did edit 800+ files ;)

**Comment by wojsmol** [comment]
@brianteeman last good commit is https://github.com/joomla/joomla-cms/pull/10659/commits/d1796dcec0b78e0aba0c5899ad03bbe3ec6f8a02

**Comment by wojsmol** [comment]
@JoomliC @andrepereiradasilva Plese do a final check.
@brianteeman Please remove Unit/System Tests label.

**Comment by brianteeman** [comment]
I've removed the label

**Comment by JoomliC** [comment]
@wojsmol i will redo a full review tomorrow ;-)

**Comment by wojsmol** [comment]
@JoomliC @andrepereiradasilva This branch contains conflicts again, check it out after work.

File: administrator/components/com_content/config.xml [hunk 35]
```diff
@@ -609,96 +672,116 @@
...
+			<option value="0">JGLOBAL_DOWN</option>
+			<option value="1">JGLOBAL_ACROSS</option>
 		</field>
 
-		<field name="spacer1"
+		<field
+			name="spacer1"
 			type="spacer"
 			hr="true"
+<<<<<<< d289eadb1b93728dbccb48f4a235f8c53c2839fe
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: Maybe a bit of clean here ? ;)

File: administrator/components/com_content/config.xml [hunk 36]
```diff
@@ -609,96 +672,116 @@
...
-				default="0"
-				description="JGLOBAL_SHOW_SUBCATEGORY_CONTENT_DESC"
-				label="JGLOBAL_SHOW_SUBCATEGORY_CONTENT_LABEL"
+		<field
+			name="subcategories"
+			type="spacer"
+			label="JGLOBAL_SUBSLIDER_BLOG_EXTENDED_LABEL"
+			class="spacer"
+		/>
+>>>>>>> XML CS - backend
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: Clean here too...

File: administrator/components/com_content/config.xml [hunk 37]
```diff
@@ -609,96 +672,116 @@
...
 		</field>
 
-		<field name="spacer1"
+		<field
+			name="spacer1"
 			type="spacer"
 			hr="true"
+<<<<<<< d289eadb1b93728dbccb48f4a235f8c53c2839fe
 			/>
+=======
```
Comment (JoomliC) [inline administrator/components/com_content/config.xml]: This line too...

**Comment by JoomliC** [comment]
@wojmol did a full review of changes, and it's ok for me!
Just the 3 mentionned by note lines to be removed ;-)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

**Comment by wojsmol** [comment]
@JoomliC conflict markers removed in https://github.com/joomla/joomla-cms/pull/10659/commits/289975cb04befa7f5c76c3e3ea496dd1535f18c8
@brianteeman Pleaseremove Language Change label - added by @joomla-cms-bot as a effrct of bad rebase.

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 289975cb04befa7f5c76c3e3ea496dd1535f18c8<br /><br />Code review + Patch applied
All seems good!

:+1: 

<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10659">issues.joomla.org/joomla-cms/10659</a>.</sub>

**Comment by wojsmol** [comment]
Small rebase after https://github.com/joomla/joomla-cms/pull/11483

**Comment by wojsmol** [comment]
rebase after recent changes

**Comment by Quy** [comment]
Is this outdated?

**Comment by wojsmol** [comment]
@Quy Tgis is very oudated  especially after recent PR's from @brianteeman.

**Comment by franz-wohlkoenig** [comment]
@wojsmol should this PR then closed?

**Comment by wojsmol** [comment]
When I have several free hours, I will go over com_content again and create a new PR if needed.

**Pull request closed without merging**